DBM-Naxx/MilitaryQuarter/Gothik.lua

```diff
@@ -76,21 +76,24 @@ local function getWaveString(wave)
 		return L.WarningWave3:format(unpack(waveInfo))
 	end
 end
-
+--]]
 function mod:OnCombatStart(delay)
 	if mod:IsDifficulty("heroic25") then
 		waves = wavesHeroic
+		phase2Timer = 300
 	else
 		waves = wavesNormal
+		phase2Timer = 270
```

Comment:
There needs to be 'local phase2Timer' at the top of the files with the rest of variables; current code introduces a global variable - undesired.